login.php

```diff
@@ -107,30 +133,25 @@
                      <div class="column2">
 					  
 						<center>
-							<input type="submit" value="submit">
+							<input type="submit" value="Submit">
 						</center>
 					  
                   	</div>
 					  
                   </div>
 
                </form>
-				
-               <br><br><br>
 			 
-               <center>
```

Comment:
Removed the link here as users will not be able to use it unless they are an admin